skype/skype-ps/skype/New-CsHostedVoicemailPolicy.md

```diff
@@ -217,32 +201,6 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -Tenant
-
-Globally unique identifier (GUID) of the Skype for Business Online tenant account for which the new hosted voicemail policy is being created.
-For example:
-
-`-Tenant "38aad667-af54-4397-aaa7-e94c79ec2308"`
-
-You can return the tenant ID for each of your tenants by running this command:
-
-`Get-CsTenant | Select-Object DisplayName, TenantID`
-
-
-
-```yaml
-Type: Guid
-Parameter Sets: (All)
-Aliases: 
-Applicable: Lync Server 2013, Skype for Business Server 2015, Skype for Business Server 2019
```

Comment:
Other changes need to be undone. Remove the appropriate Applicable version(s) from the `-Tenant` parameter.